### DISCUSSION TITLE
razer synapse for mac

### DISCUSSION THREAD

**Discussion by sermii1** [community/community#31964] [Other Feature Feedback, Questions, & Ideas]
when downloading synapse razer on a poppy, when I open it, an error pops up. A JavaScript error occurred in the main process. what should I do?

**Comment by MathiasReker** [comment]
search for the error on Google. In most cases, someone else has had the same problem.

**Comment by AndreSubbota** [comment]
Hello,
i tried the Razer for Mac OS from Ken Chen. My problem is, that i am using a Basilisk V3 Pro mice and it doesn´t work with the software. The Software doesn´t recognize it and i can´t change the Chroma lighting on it. Will there be an update in the future, which will support the Basilisk V3 Pro?

Thank you

**Reply by ebndev** [reply]
Hi @AndreSubbota,

Thanks for being a part of the GitHub Community, we're glad you're here!

It looks like you're looking for support from [Razer](https://mysupport.razer.com/). You might want to try asking for help somewhere that focuses on their products or inquiring on their support page.

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place.

Best of luck!
